src/main/groovy/org/transmartproject/batch/gwas/metadata/GwasMetadataStore.groovy

```diff
@@ -0,0 +1,67 @@
+package org.transmartproject.batch.gwas.metadata
+
+import com.google.common.collect.Lists
+import groovy.util.logging.Slf4j
+import org.springframework.batch.core.ExitStatus
+import org.springframework.batch.core.StepExecution
+import org.springframework.batch.core.annotation.AfterStep
+import org.springframework.batch.core.configuration.annotation.JobScope
+import org.springframework.batch.item.ExecutionContext
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+
+import javax.annotation.PostConstruct
+
+/**
+ * The place where the {@link GwasMetadataEntry} objects are stored.
+ */
+@Component
+@JobScope
+@Slf4j
+class GwasMetadataStore {
```

Comment:
Does it make sense to have whole class instead of just list of entries for this purpose?